accessible.go

```diff
@@ -74,3 +75,16 @@ type UIAutomationLegacyAccessiblePatternVtbl struct {
 func (lap *UIAutomationLegacyAccessiblePattern) VTable() *UIAutomationLegacyAccessiblePatternVtbl {
 	return (*UIAutomationLegacyAccessiblePatternVtbl)(unsafe.Pointer(lap.RawVTable))
 }
+
+func (lap *UIAutomationLegacyAccessiblePattern) DoDefaultAction() error {
+	hr, _, _ := syscall.SyscallN(
+		lap.VTable().DoDefaultAction,
+		uintptr(unsafe.Pointer(lap)),
+	)
+
+	if hr != ole.S_OK {
```

Comment:
Just for consistency with other files:

```suggestion
	if hr != 0 {
```

Reply:
If you think the `0` value is unclear, feel free to open an issue and I can update it everywhere it's used